README.md

```diff
@@ -16,6 +15,8 @@ It features:
 * Testing your project.
 * Benchmarking your project.
 
+`stack` replaces `stackage-cli`. For more details on the motivation for the creation of `stack`, a comparison of `stack` to other tools, and the roadmap for the future of `stack', [see this document]().
```

Comment:
I'm not sure stackage-cli was ever popular enough for this message to warrant being front and center in the README, I think it will end up leading to more confusion around "what's stackage-cli?"